dev-libs/bglibs/files/bglibs-2.04-fix-feature-tests.patch

```diff
@@ -0,0 +1,369 @@
+@@ -1,7 +1,7 @@
+ #include <sys/types.h>
+ #include <unistd.h>
+ 
+-int main()
++int main(void)
+ {
+   getpeereid();
+ }
+diff -ruN a/sys/trypeercred.c b/sys/trypeercred.c
+--- a/sys/trypeercred.c	2024-04-06 16:39:11.932631664 -0000
++++ b/sys/trypeercred.c	2024-04-06 16:40:00.972334630 -0000
+@@ -2,7 +2,7 @@
+ #include <sys/types.h>
+ #include <sys/socket.h>
+ 
+-int main()
++int main(void)
+ {
+   struct ucred peer;
+   int optlen = sizeof(peer);
+diff -ruN a/sys/trypoll.c b/sys/trypoll.c
+--- a/sys/trypoll.c	2024-04-06 16:39:11.945631585 -0000
++++ b/sys/trypoll.c	2024-04-06 16:40:04.380313988 -0000
+@@ -3,7 +3,7 @@
+ #include <poll.h>
+ #include <unistd.h>
+ 
+-int main()
++int main(void)
+ {
+   struct pollfd x;
+ 
+diff -ruN a/sys/trysetenv.c b/sys/trysetenv.c
+--- a/sys/trysetenv.c	2024-04-06 16:39:11.935631646 -0000
++++ b/sys/trysetenv.c	2024-04-06 16:40:08.044291795 -0000
+@@ -1,6 +1,6 @@
+ #include <stdlib.h>
+ 
+-int main()
++int main(void)
+ {
+   setenv("FOO", "bar", 1);
+   return 0;
+diff -ruN a/sys/trysigaction.c b/sys/trysigaction.c
+--- a/sys/trysigaction.c	2024-04-06 16:39:11.945631585 -0000
++++ b/sys/trysigaction.c	2024-04-06 16:40:13.676257682 -0000
+@@ -1,6 +1,6 @@
+ #include <signal.h>
+ 
+-int main()
++int main(void)
+ {
+   struct sigaction sa;
+   sa.sa_handler = 0;
+diff -ruN a/sys/trysigprocmask.c b/sys/trysigprocmask.c
+--- a/sys/trysigprocmask.c	2024-04-06 16:39:11.944631591 -0000
++++ b/sys/trysigprocmask.c	2024-04-06 16:40:18.363229293 -0000
+@@ -1,6 +1,6 @@
+ #include <signal.h>
+ 
+-int main()
++int main(void)
+ {
+   sigset_t ss;
+  
+diff -ruN a/sys/tryspnam.c b/sys/tryspnam.c
+--- a/sys/tryspnam.c	2024-04-06 16:39:11.945631585 -0000
++++ b/sys/tryspnam.c	2024-04-06 16:40:21.124212570 -0000
+@@ -1,7 +1,7 @@
+ #include <shadow.h>
+ #include <stdio.h>
+ 
+-int main()
++int main(void)
+ {
+   struct spwd *spw;
+ 
+diff -ruN a/sys/trysysselect.c b/sys/trysysselect.c
+--- a/sys/trysysselect.c	2024-04-06 16:39:11.933631658 -0000
++++ b/sys/trysysselect.c	2024-04-06 16:40:24.052194835 -0000
+@@ -2,7 +2,7 @@
+ #include <sys/time.h>
+ #include <sys/select.h> /* SVR4 silliness */
+ 
+-void foo()
++void foo(void)
+ {
+   ;
+ }
+diff -ruN a/sys/tryulong32.c b/sys/tryulong32.c
+--- a/sys/tryulong32.c	2024-04-06 16:39:11.945631585 -0000
++++ b/sys/tryulong32.c	2024-04-06 16:40:29.116164162 -0000
+@@ -1,6 +1,6 @@
+ #include <unistd.h>
+ 
+-int main()
++int main(void)
+ {
+   unsigned long u;
+   u = 1;
+diff -ruN a/sys/tryulong64.c b/sys/tryulong64.c
+--- a/sys/tryulong64.c	2024-04-06 16:39:11.945631585 -0000
++++ b/sys/tryulong64.c	2024-04-06 16:40:32.140145846 -0000
+@@ -1,6 +1,6 @@
+ #include <unistd.h>
+ 
+-int main()
++int main(void)
+ {
+   unsigned long u;
+   u = 1;
+diff -ruN a/sys/tryuserpw.c b/sys/tryuserpw.c
+--- a/sys/tryuserpw.c	2024-04-06 16:39:11.945631585 -0000
++++ b/sys/tryuserpw.c	2024-04-06 16:40:37.180115319 -0000
+@@ -1,6 +1,6 @@
+ #include <userpw.h>
+ 
+-int main()
++int main(void)
```

Comment:
I kinda accidentally did it in two places and haven't consolidated the patch.

Reply:
Yea, I see what you mean, you should be able to just squash the patches with a git rebase.